src/routes/index.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import { Switch, Route } from 'react-router-dom';
+
+import { Login } from '../screens/Login';
+
+const RouteList = () => {
+  return (
+    <Switch>
+      <Route exact path="/" component={Login}></Route>
```

Comment:
@noor-ahmed Between `{Login}` and `{ Login }`, which one should we go with it? 😅

Reply:
`{ Login }` is 👌